apps/console/src/features/identity-providers/components/forms/authenticators/microsoft-authenticator-form.tsx

```diff
@@ -267,7 +275,7 @@ export const MicrosoftAuthenticatorForm: FunctionComponent<MicrosoftAuthenticato
         if (scope === IdentityProviderManagementConstants.MICROSOFT_SCOPE_DICTIONARY.PROFILE) {
             return {
                 description: t("console:develop.features.authenticationProvider.forms" +
-                    ".authenticatorSettings.microsoft.scopes.list.profile.description"),
+                    ".authenticatorSettings.microsoft.Scopes.list.profile.description"),
```

Comment:
Why have we used `Scopes` instead of `scopes`. Usually, the i18n should follow camel-case. Check in other places as well.